apps/www/_blog/2023-08-07-hugging-face-supabase.mdx

```diff
@@ -0,0 +1,237 @@
+---
+title: 'Hugging Face is now supported in Supabase'
+description: "We've added support Hugging Face support in our Python Vector Client and Edge Functions."
+launchweek: 8
+tags:
+  - launch-week
+  - announcements
+date: '2023-08-07'
+toc_depth: 3
+author: gregnr,oli_rice
+image: launch-week-8/day-1/hugging-face-supabase-og.jpg
+thumb: launch-week-8/day-1/hugging-face-supabase-thumb.jpg
+---
+
+Since launching our [Vector Toolkit](https://supabase.com/vector) a few months ago, the number of AI applications on Supabase has grown - a lot. Hundreds of new databases every week are using [pgvector](https://github.com/pgvector/pgvector).
+
+98% of those applications use OpenAI to store embeddings using their `text-embedding-ada-002` model. While OpenAI is easy, it’s not open source, which means it can’t be self-hosted.
```

Comment:
```suggestion
98% of those applications use OpenAI to store embeddings using the `text-embedding-ada-002` model. While OpenAI is easy, it’s not open source, which means it can’t be self-hosted.
```